sos/report/plugins/openvswitch.py

```diff
@@ -146,6 +146,11 @@ def setup(self):
             f"{self.actl} dpdk/get-malloc-stats",
             # Capture dpdk mempool info
             f"{self.actl} netdev-dpdk/get-mempool-info"
+            # Capture OVS-DOCA info
+            f"{self.actl} dpdk/get-mempool-stats",
```

Comment:
i think that in this case it will run the command regardless of upstream or doca and will return an error. 
Can you please check that? and if return an error then you should probably add a check before running those commands.

Reply:
done